index.bs

```diff
@@ -1516,6 +1527,8 @@ When this operation is invoked, the authenticator must perform the following pro
     so, return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
 - If the |requireResidentKey| flag is set to |true| and the authenticator cannot store a [=Client-side-resident Credential
     Private Key=], return an error code equivalent to "{{ConstraintError}}" and terminate the operation.
+- If the |requireRateLimiting| flag is set to |true| and the authenticator cannot support [=Rate Limiting=], return an error code 
```

Comment:
@equalsJeffH 

I am sorry - I don't understand this response.  One one hand, you have stated the following:  "Given that rate-limiting of user verification attempts is already a requirement for FIDO-certified authenticators, I wonder whether this is needed.".  Yet here you seem to be stating that the webauthn specification should not normatively require authenticators to be FIDO certified.

What would be your preferred  approach (in terms of specification text) to ensuring that rate limiting be required across all webauthn authenticators?